database/migrations/2016_06_07_153443_create_smart_lives_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateSmartLivesTable extends Migration
```

Comment:
yes i know, actually i planned to change the table later when i'll have a clearer view of what will be the content there. I did this way just cos for me they are just two onthologically different things.. since they will be two different groups of topic in the IDM schemas i thought they should have different models